src/site/content/en/blog/proxx-1/sourcemap.html

```diff
@@ -0,0 +1,210 @@
+<style>
```

Comment:
Are you able to remove this file? It looks like it was inlined directly into the article instead.

Reply:
Turned this file into an SVG to address the above issue.